### ISSUE TITLE
Screenshot capture text is included in the screenshot

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jennorby/mupen64plus#290]
```
Describe your system:
 - Linux distribution: Ubuntu 9.04
 - Machine type: 64-bit
 - Mupen64Plus version: 1.5
 - Plugins used: Rice's Video Plugin 1.5, JttL's SDL Audio 1.5, blight's
SDL input plugin 0.0.10.0, and Hacktarus/Azimer hle rsp plugin

Describe the problem:
When taking multiple screenshots in a row and "On screen display enabled"
is checked, the screenshot capture text will appear in the captured PNGs. 
I'm not familiar with how the graphics work, but my two proposed solutions
are either display the capture text on a different layer of video and
exclude that from the screenshot, or remove any on screen text right before
capturing.
```

Original issue reported on code.google.com by `hansamu...@gmail.com` on 22 Oct 2009 at 4:24

Attachments:
- [beetle_adventure_rac-097.png](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-290/comment-0/beetle_adventure_rac-097.png)

**Comment by GoogleCodeExporter** [comment]
```
I looked at this problem and it may not be easy to fix.  I actually wrote the
screenshot code to avoid this problem - it first takes a screenshot after 
rendering
the display, and then it paints the OSD text.  If you take a lot of screenshots 
in a
row, you will see that this bug only shows up in about half of them.  That's 
because
some or all games actually update at 30fps, though the render function is 
called once
per field, ie 60 fps, because this was intended to be used on NTSC interlaced 
TVs.
During the second rendering period the ROM code does not clear the screen or 
re-draw
anything; it just lets the hardware display the other field from the same 
(previous)
frame.  There are several different ways that I can think to fix this bug, but 
they
require significant testing because they may introduce other bugs.
```

Original comment by `richard...@gmail.com` on 14 Jan 2010 at 5:00
- Changed state: **Accepted**
- Added labels: **Component-Core**, **Type-Visual**

**Comment by GoogleCodeExporter** [comment]
Original comment by `richard...@gmail.com` on 22 May 2011 at 3:56
- Added labels: **Milestone-2.0**

**Comment by GoogleCodeExporter** [comment]
```
Fixed by updating Video plugin API to 2.1.0
```

Original comment by `richard...@gmail.com` on 4 Mar 2012 at 11:17
- Changed state: **Fixed**
